src/main/java/TechJobs.java

```diff
@@ -119,7 +117,15 @@ private static String getUserSelection(String menuHeader, HashMap<String, String
 
     // Print a list of jobs
     private static void printJobs(ArrayList<HashMap<String, String>> someJobs) {
+        for (int i = 0; i < someJobs.size(); i++) {
+            System.out.println("*****");
+            for (Map.Entry<String,String> job : someJobs.get(i).entrySet()) {
+                System.out.println( job.getKey()+": "+ job.getValue());
+
+            }
+            System.out.println("*****");
+            System.out.println("\n");
```

Comment:
Your tests are failing because of the new lines that are being added. Try to use System.out.print() instead of System.out.println()